replica/backup.go

```diff
@@ -189,6 +189,17 @@ type Hole struct {
 
 var HoleCreatorChan = make(chan Hole, 1000000)
 
+func holeDrainer() {
+	types.DrainOps = types.DrainStart
+	HoleCreatorChan <- Hole{}
```

Comment:
lets handle empty structure in CreateHoles()

Reply:
@vishnuitta not required, since we are setting types.DrainOps = types.DrainDone draining it completely, so no empty structure will be there.